src/lib/api/agent.ts

```diff
@@ -36,6 +36,34 @@ export async function add(dept_id: Dept['dept_id'], uid: User['user_id'], head:
     }
 }
 
+/** Removes an {@linkcode Agent} from the {@linkcode Dept}. Returns `true` if successful, `false` otherwise. */
```

Comment:
Actually, we return whether the deleted department agent was a head or not. Otherwise, it is `null` if not found.